core/src/main/java/be/isach/ultracosmetics/cosmetics/particleeffects/ParticleEffect.java

```diff
@@ -56,9 +56,12 @@ && getOwner().getCurrentParticleEffect() == this) {
                     if (isMoving()) {
                         if (getType().getEffect() == Particles.REDSTONE) {
                             if (!ignoreMove) {
-                                Particles.OrdinaryColor color = getType().getConfigName().equals("angelwings")
-                                        ? new Particles.OrdinaryColor(255, 255, 255)
-                                        : new Particles.OrdinaryColor(255, 0, 0);
+                                Particles.OrdinaryColor color = new Particles.OrdinaryColor(255, 0, 0);
+                                if (getType().getConfigName().equals("AngelWings")) {

```

Comment:
After updating this logic, the check `getType().getConfigName().equals("angelwings")` was always false because the config name was `AngelWings`. Should I leave this update here?

Reply:
Yes, that looks good